### PR TITLE
Describe request in terms of fetch.

### LINKED ISSUE #51: Define functionality in terms of `fetch()` (or otherwise make sure that caching / storage policy matches the rest of the platform).
This issue is part of the PING privacy review w3cping/privacy-request#61

Currently the spec seems to define the document as the lifetime for the results of IFT responses.

The document is an uncommon privacy boundary on the Web; more commonly cache boundaries are defined in terms of eTLD+1, profile, etc. https://github.com/privacycg/storage-partitioning has more information about the effort and goals of partitioning storage and caches.

I think the best fix here would be to define the functionality here in terms of `fetch()`, in which case the proposal would "get" aligning with storage partitioning work "for free." If defining in terms of `fetch()` won't work though, I suggest working with the https://github.com/privacycg/storage-partitioning folks to figure out how to best align this functionality with the privacy-focused partitioning work already happening.

### PR DESCRIPTION
**Pull request by garretrieger**
Fixes #51

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/IFT/pull/66.html" title="Last updated on Dec 14, 2021, 12:15 AM UTC (2835001)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/IFT/66/e3aa303...2835001.html" title="Last updated on Dec 14, 2021, 12:15 AM UTC (2835001)">Diff</a>

### CODE REVIEW

**Comment by pes10k** [comment]
lgtm, thanks!

**Review by vlevantovsky — APPROVED** [review]
LGTM, thanks!

File: Overview.bs [hunk 1]
```diff
@@ -676,19 +677,25 @@ transferred:
...
+A client extends its font subset to cover additional codepoints by making requests to a Patch Subset
+server. The request requirements are described here in terms of a fetch according to
+"4. Fetching" in [[!fetch]]. If the implementing user agent does not support fetch, then the request
+should be made using an equivalent HTTP request.
 
-*  If sent as a POST request the post body will be a single
-    <a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR.
+* The request <strong>method</strong> must be either "GET" or "POST".
+
+* The request <strong>destination</strong> should be "font".
```
Comment (jyasskin) [inline Overview.bs:687]: Why "should" instead of "must"?

File: Overview.bs [hunk 2]
```diff
@@ -676,19 +677,25 @@ transferred:
...
-A client extends its font subset to cover additional codepoints by making HTTP requests to
-a Patch Subset server. The HTTP request must use either the GET or POST method:
+A client extends its font subset to cover additional codepoints by making requests to a Patch Subset
+server. The request requirements are described here in terms of a fetch according to
+"4. Fetching" in [[!fetch]]. If the implementing user agent does not support fetch, then the request
+should be made using an equivalent HTTP request.
 
-*  If sent as a POST request the post body will be a single
-    <a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR.
+* The request <strong>method</strong> must be either "GET" or "POST".
```
Comment (jyasskin) [inline Overview.bs:685]: This is Bikeshed (https://tabatkins.github.io/bikeshed/), so you should use its cross-spec referencing abilities:

```
* The request [=request/method=] must be either "GET" or "POST".
```

**Pull request merged**